### PR TITLE
Functions for hashing and hashcomparing password

### PR DESCRIPTION
**Pull request by harry-harish**
Create a folder 'utils' in the root project folder and add a file 'security.js' if does not exist already.
In the file security.js, add two utility functions (which can be used in other files)-
1. getHashedValue(String str) - Returns hashed value of str
2. checkHash(str, hash) - Returns true if hashed value of str equals hash, else returns false.

### CODE REVIEW

File: utils/security.js [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+ *
+ *
+ * @param {*} str
+ * @returns {*} hash
+ */
+exports.getHashedValue = async (str) => {
+    let hash = await bcrypt.hash(str, BCRYPT_SALT_ROUNDS);
+    if(hash){
+        return hash;
+    };
```
[the remark this answers is not in the document]
Comment (saumitra2810) [reply utils/security.js]: Think we need to throw Error.

**Comment by shivamproso** [comment]
@saumitra2810 yes, the test cases seem important here. @harry-harish Can you incorporate some unit tests for the written functions?
this might help -  https://codeburst.io/how-to-test-javascript-with-mocha-the-basics-80132324752e

File: test/security.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const expect = require("chai").expect;
```
Comment (shivamproso) [inline test/security.test.js]: can we follow a naming convention for test file as filename_test.js . So, can we rename this file to security_test.js?

Comment (harry-harish) [reply test/security.test.js]: Yes.. i will change the file name

**Pull request merged**